backend/controllers/cellMove/selectLocation.js

```diff
@@ -51,7 +53,8 @@ module.exports = ({ elite2Api, logError }) => async (req, res) => {
       nonAssociations.nonAssociations.some(
         nonAssociation =>
           nonAssociation.offenderNonAssociation.agencyDescription.toLowerCase() ===
-          prisonerDetails.assignedLivingUnit.agencyName.toLowerCase()
+            prisonerDetails.assignedLivingUnit.agencyName.toLowerCase() &&
```

Comment:
Sure it's fine but can offenderNonAssociation and assignedLivingUnit be null?

Reply:
Probably :D The API wouldn't return the nonAssociation record without the corresponding offender, and the offender is always serialised the same.